lib/node_modules/@stdlib/math/base/complex/divide/docs/repl.txt

```diff
@@ -0,0 +1,40 @@
+
+{{alias}}( [out,] re1, im1, re2, im2 )
+    Divides two complex numbers.
+
+    Parameters
+    ----------
+    out: Array|TypedArray|Object
```

Comment:
Indeed! That was a partial motivation for, e.g., adding JSDoc annotations to ESLint rules (e.g., https://github.com/stdlib-js/stdlib/blob/develop/etc/eslint/rules/best_practices.js). And the JavaScript style guide has TODOs to point to the various ESLint rules. But alas, hooking everything all together is still a work in progress. :)